.github/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 <h3>Improvements</h3>
 
+- Added the `ControlledPhaseShift` gate as well as the `QFT` operation for applying quantum Fourier
+  transforms.
+  [(#1064)](https://github.com/PennyLaneAI/pennylane/pull/1064)
```

Comment:
Actually, I would add these to new features, since they are very visible new features.

Reply:
Nice, done!